build/Targets/TestProgram.cs

```diff
@@ -2,16 +2,31 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using Microsoft.NET.TestFramework;
 
 class Program
 {
     public static int Main(string[] args)
     {
-        var newArgs = args.ToList();
-        newArgs.Insert(0, typeof(Program).Assembly.Location);
+        var newArgs = TestCommandLine.HandleCommandLine(args, out bool showHelp);
```

Comment:
I think `Environment.GetCommandLineArgs()` achieves the same thing and would work here.